manifests/templates/cicd/code-scan.yaml

```diff
@@ -0,0 +1,37 @@
+- apiVersion: cyclone.io/v1alpha1
+  kind: Stage
+  metadata:
+    name: sonarqube-scan-template
+    labels:
+      cyclone.io/stage-template: "true"
+      cyclone.io/scene: cicd
+      cyclone.io/template-kind: code-scan
+    annotations:
+      cyclone.io/description: >
+        Source code statistically scan with SonarQube
+  spec:
+    pod:
+      inputs:
+        arguments:
+        - name: sonarqube
```

Comment:
Where will use this argument?

Reply:
This argument is used for Users to choose a sonarqube instance from Integration.